Streetcode/Streetcode.BLL/DTO/AdditionalContent/Subtitles/SubtitleDTO.cs

```diff
@@ -8,8 +8,8 @@ public class SubtitleDTO
     public SubtitleStatusDTO SubtitleStatus { get; set; }
     public string FirstName { get; set; }
     public string LastName { get; set; }
-    public string Description { get; set; }
-    public UrlDTO Url { get; set; }
+    public string? Description { get; set; }
+    public UrlDTO? Url { get; set; }
```

Comment:
same for navigation properties
do we need all-time return related Dtos in Api?

Reply:
I don't really understand what you mean